docs/cms/developers/quick-start/index.md

```diff
@@ -2,9 +2,19 @@
 
 ## Introduction
 
-This guide explains how to update a simple Howlite component. Before you complete this guide, we recommend completing the basic [quick start guide](/cms/quick-start/) first so that you understand basic WebSight CMS concepts.
+!!! info "Important notice"
+
+    Before you complete this guide, we recommend completing the basic [quick start guide](/cms/quick-start/) first so that you understand basic WebSight CMS concepts.
+
+This guide uses [starter](https://github.com/websight-io/starter) - our official project which demonstrates variuous functionality of Websight CMS and pre-configures for you the following features:
+
+- Demo site project _Luna_
+- Integration with [Howlite](https://github.com/websight-io/howlite) library
```

Comment:
Integration is not the right wording here

Reply:
changed this to "Howlite components library"